app/javascript/src/components/ItemList/ItemListItem.jsx

```diff
@@ -13,6 +13,9 @@ export const ItemListItem = ({ item, children }) => {
     tags: item.article_tags || item.reactable_tags,
   };
 
+  // update readingTime to 1 min if the reading time is less than 1 min
+  adaptedItem.readingTime = Math.max(1, adaptedItem.readingTime);
```

Comment:
Hi @akshayymahajan, `Math.max` is a great idea but there's a catch: if for any reason `readingTime` is `undefined` this will return `NaN`.

I think the best bet is: `Math.max(1, adaptedItem.readingTime || null)` because `Math.max(1, null)` returns 1 correctly.

It shouldn't happen in reality but it's better to be safe :D